views/frontend/post.php

```diff
@@ -1,71 +1,68 @@
-<?php
-    $post = $model_article->get_article_blog();
-    if($post == false){
-        header("Location:/page-erreur");
-    }else{
-?>
-    
-    <div class="row center">
+<div class="row center">
         <div class=" row ">
-            <img class="responsive-img" src="/public/img/posts/<?= $post->image ?>" alt="<?= $post->title ?>"/>
+            <img class="responsive-img" src="/public/img/posts/<?= $post->getImage() ?>" alt="<?= $post->getTitle() ?>"/>
         </div>
     </div>
     
     <div class="container">
-            <h1><?= $post->title ?></h1>
-            <h6>Par <?= $post->name ?> le <?= date("d/m/Y à H:i", strtotime($post->date)) ?></h6>
-            <p><?= nl2br($post->content); ?></p>
+        <h1><?= $post->getTitle() ?></h1>
+        <h6>Par <?= $admin->getName() ?> le <?= date("d/m/Y à H:i", strtotime($post->getDate())) ?></h6>
+        <p><?= nl2br($post->getContent()); ?></p>
+        <hr>
+        <h4>Commentaires:</h4>
+        <?php
+            if($responses != false){
+                foreach($responses as $response){
+                    ?>
+                        <blockquote>
+                             <strong><?= $response->getName() ?> (<?= date("d/m/Y", strtotime($response->getDate())) ?>)</strong>
+                             <p><?= nl2br($response->getComment()); ?></p>
+                         </blockquote>
+                    <?php
+                 }
+             }else{ 
+                 echo "Aucun commentaire n'a été publié... Soyez le premier!";
+             }
+        ?>
+        <h4>Commenter:</h4>
 <?php
-}
-?>
-
-            <hr>
-            <h4>Commentaires:</h4>
-            <?php
-                $responses = $model_comment->get_comments_blog();
-                if($responses != false){
-                    foreach($responses as $response){
-                        ?>
-                            <blockquote>
-                                <strong><?= $response->name ?> (<?= date("d/m/Y", strtotime($response->date)) ?>)</strong>
-                                <p><?= nl2br($response->comment); ?></p>
-                            </blockquote>
-                        <?php
-                    }
-                }else{ 
-                    echo "Aucun commentaire n'a été publié... Soyez le premier!";
+if(!empty($errors)){
+                ?>
+                    <div class="card red">
+                        <div class="card-content white-text">
+                            <?php
+                                foreach($errors as $error){
+                                    echo $error."</br>";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)